packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/TimeFilter/Tabs/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { connect, ConnectedProps } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import styled from 'styled-components'
+
+import { TabMenu, TabMenuItem } from 'blockchain-info-components'
+import { actions } from 'data'
+import { media } from 'services/styles'
+
+const CustomTabMenu = styled(TabMenu)`
+  margin: 0 24px;
+  ${media.atLeastTabletL`
+    margin-right: 24px;
+    margin-bottom: 24px;
+  `}
+`
+
+const TabsContainer = ({
+  currentTab,
+  handleClick
+}: {
+  currentTab: string
+  handleClick: (time: 'day' | 'week' | 'month' | 'year' | 'all') => void
```

Comment:
(minor) this might be moved to be type 

```
type timeStatuses = 'day' | 'week' | 'month' | 'year' | 'all'
```

Reply:
or even better enum so you may reuse it all way down inside `selected` check etc..

Reply:
Done!